docs/releases/1.18-NOTES.md

```diff
@@ -6,6 +6,8 @@
 
 * Terraform users on AWS may need to rename some resources in their state file in order to prepare for Terraform 0.12 support. See Required Actions below.
 
+* Lyft CNI plugin default subnet tags changed from from `Type: pod` to `KubernetesCluster: myclustername.mydns.io`.
```

Comment:
Maybe we should describe the action a user would want to take if they're affected by this.

`Subnets intended for pod IP use will need to be tagged with this new tag, and additional tag filters may need to be added to the cluster spec in order to achieve the desired set of subnets.`

or something along those lines.

Reply:
I think should be ok now.